Bonsai.Configuration/PackageConfigurationUpdater.cs

```diff
@@ -69,6 +69,36 @@ string GetRelativePath(string path)
             return PathUtility.GetPathWithDirectorySeparator(relativeUri.ToString());
         }
 
+        static string CombinePath(string path1, string path2)
+        {
+            return PathUtility.GetPathWithForwardSlashes(Path.Combine(path1, path2));
+        }
+
+        static PackageConfiguration NormalizePathSeparators(PackageConfiguration configuration)
```

Comment:
This method feels like it belongs on `PackageConfiguration` itself. (Maybe as `WithNormalizedPathSeparators`.)

As it stands right now, there's a big risk of someone overlooking this method and not knowing that it needs to be updated if they were adding a field to `PackageConfiguration`.

Reply:
This would also make it more reasonable to unit test this behavior in isolation if you have time for a quick test.

Reply:
Point taken about unit testing, but the rationale behind it is that we actually don't want to normalize all paths in the configuration (see below), and it also might mislead developers that this is somehow a desirable behavior to do, whereas actually there is only a limited set of cases for it.

Also in my case it helps to know that all responsibility for path normalization is centered in a single class. More details in the below comment, but happy to revisit this.

Reply:
Looking at it again I agree, especially since normalizing the `ConfigurationFile` is definitely actually a bad thing.